source/Xunit.Gherkin.Quick.UnitTests/FeatureClassTests.cs

```diff
@@ -47,58 +47,118 @@ private sealed class FeatureWithFilePath : Feature
         }
 
         [Fact]
-        public void ExtractScenario_Extracts_Scenario()
+        public void ExtractScenario_Extracts_Scenario_Without_Background()
         {
             //arrange.
             var scenarioName = "some scenario name 123";
             var featureInstance = new FeatureWithMatchingScenarioStepsToExtract();
             var sut = FeatureClass.FromFeatureInstance(featureInstance);
-
-            //act.
-            var scenario = sut.ExtractScenario(CreateGherkinDocument(scenarioName,
-                new string[] 
-                {
-                    "Given " + FeatureWithMatchingScenarioStepsToExtract.ScenarioStep1Text.Replace(@"(\d+)", "12", StringComparison.InvariantCultureIgnoreCase),
-                    "And " + FeatureWithMatchingScenarioStepsToExtract.ScenarioStep2Text.Replace(@"(\d+)", "15", StringComparison.InvariantCultureIgnoreCase),
-                    "When " + FeatureWithMatchingScenarioStepsToExtract.ScenarioStep3Text,
-                    "Then " + FeatureWithMatchingScenarioStepsToExtract.ScenarioStep4Text.Replace(@"(\d+)", "27", StringComparison.InvariantCultureIgnoreCase)
-                }).Feature.Children.First() as Gherkin.Ast.Scenario);
+			var gherkinScenario = CreateGherkinDocument(scenarioName,
```

Comment:
Can you please use the same indentation as the rest of the code? Otherwise this newly added code looks strangely off the alignment.
Applies to most of the code you wrote. All they somehow are using different indentation size than the existing code.
As a standard, this whole codebase uses 4 spaces to denote each level of indentation. I think you are using 8 instead of 4.

Reply:
Hmm.. Not sure how this has happened. I don't have indentation issues showing in VS Code or Visual Studio, and I have indents set to 4. So I am not sure how or where this is coming from.

Reply:
Okay. Can you at least shorten the indent twice? Does not matter if the VS code shows it. If it's twice shorter for the new lines of code that you added, it must match the existing lines.
If that does not help, then I will need to go through the code after you merge to fix the indents.
That's also fine with me, but would be better if I don't need to hunt down these.